public/app/features/migrate-to-cloud/onprem/Page.tsx

```diff
@@ -1,32 +1,75 @@
 import { skipToken } from '@reduxjs/toolkit/query/react';
 import React, { useState } from 'react';
 
-import { Button, Stack, Text } from '@grafana/ui';
-import { Trans } from 'app/core/internationalization';
+import { Alert, Box, Button, Stack } from '@grafana/ui';
+import { Trans, t } from 'app/core/internationalization';
 
-import { useGetStatusQuery, useListResourcesQuery } from '../api';
+import { useGetStatusQuery, useListResourcesQuery, useStartMigrationMutation } from '../api';
 
 import { DisconnectModal } from './DisconnectModal';
 import { EmptyState } from './EmptyState/EmptyState';
+import { MigrationInfo } from './MigrationInfo';
 import { ResourcesTable } from './ResourcesTable';
 
 export const Page = () => {
   const { data: status, isFetching } = useGetStatusQuery();
   const { data: resources } = useListResourcesQuery(status?.enabled ? undefined : skipToken);
+  const [startMigration, { isLoading: startMigrationIsLoading, isError: startMigrationIsError }] =
+    useStartMigrationMutation();
   const [isDisconnecting, setIsDisconnecting] = useState(false);
 
   if (!status?.enabled) {
     return <EmptyState />;
   }
 
+  const isBusy = isFetching || isDisconnecting || startMigrationIsLoading;
+
   return (
     <>
-      <Stack direction="column" alignItems="flex-start">
-        {status.stackURL && <Text variant="h4">{status.stackURL}</Text>}
+      <Stack direction="column" gap={4}>
+        {startMigrationIsError && (
+          <Alert
+            severity="error"
+            title={t(
+              'migrate-to-cloud.summary.error-starting-migration',
+              'There was an error starting cloud migration'
+            )}
+          />
+        )}
+
+        {status.stackURL && (
+          <Box
+            borderColor="weak"
+            borderStyle="solid"
+            padding={2}
+            display="flex"
+            gap={4}
+            alignItems="center"
+            justifyContent="space-between"
+          >
+            <Stack gap={4} alignItems="center">
```

Comment:
```suggestion
```
This Stack is not needed anymore.

Reply:
hm yeah you're technically right for the current state, I guess 😅

Reply:
i sympathise cause i know from the design this _will_ be needed fairly imminently 😄